static/css/temp

```diff
@@ -0,0 +1 @@
+
```

Comment:
@Archies11 Can you explain these temporary files added?

Reply:
I added these files while creating the folders as empty folders cant be created. i will delete them right now and provide a link gh pages link as well